kratos/solving_strategies/schemes/scheme.h

```diff
@@ -225,192 +191,185 @@ class Scheme
     {
     }
 
-
     /** Destructor.
      */
     virtual ~Scheme()
     {
     }
 
-
-    /*@} */
-    /**@name Operators
-     */
-    /*@{ */
+    ///@}
+    ///@name Operators
+    ///@{
 
     /**
-     * Clone 
+     * @brief Clone method
+     * @return The pointer of the cloned scheme 
      */
     virtual Pointer Clone()
     {
-      return Pointer( new Scheme(*this) );
+        return Kratos::make_shared<Scheme>(*this) ;
```

Comment:
uff, this is terrible... can u try to remove the Clone function to see if it still compiles?

actually one of the requests i wanted to make was to delete the copy constructor of the scheme. if we do so this wouldn't compile...

Reply:
In thgis PR or in a different PR?

Reply:
i would put the copy constructor as explicit and then fix this. (which will mean leaving the original version)

then maybe we could take the Clone out as a different PR